classes/librarian/Series.js

```diff
@@ -0,0 +1,94 @@
+import moment from 'moment';
+import escape from 'escape-regexp';
+
+import SeriesModel from 'models/Series';
+import BookModel from 'models/Book';
+
+/**
+括弧で囲まれた文字列を削除する
+@param { String } title - 書籍のタイトルデータ
+@return { String } 括弧で囲まれた文字列を削除した書籍のタイトルデータ
+**/
+function trimChar(rawStr) {
+  let trimedStr;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).